packages/stacked_cli/lib/src/commands/create/create_app_command.dart

```diff
@@ -55,16 +67,63 @@ class CreateAppCommand extends Command {
   @override
   Future<void> run() async {
     await _configService.loadConfig();
+    _cLog.stackedOutput(message: argResults!.rest.first, isBold: true);
     final appName = argResults!.rest.first;
     final appNameWithoutPath = appName.split('/').last;
     final templateType = argResults![ksTemplateType];
+    String organization = "";
+    String platforms = "";
+
+    int indexOrg = argResults!.arguments.indexOf("--org");
+    if (indexOrg != -1) {
+      organization = argResults!.arguments[indexOrg + 1];
+    }
+
+    if (indexOrg == -1) {
+      indexOrg = argResults!.arguments.indexOf("-o");
+      if (indexOrg != -1) {
+        organization = argResults!.arguments[indexOrg + 1];
+      }
+    }
```

Comment:
- The code has to be DRY (Don't Repeat Code). You have multiple lines of the exact same code in a row, 4 times in a row. 
- None of the other commands use this convention you're using to check if an option was supplied. Please make sure you understand how the other commands to it and use the same way. 
- No Magic values in our code unless there's an explicit instruction that it has to be generated in the future from the compile command. Please remove all magic values
- You don't have to check the option for both versions, again, look at the implementation of the other options. Follow along with the one that closes resemble what you want to implement and copy the structure and implementation of that.